pkg/agent/controller/egress/egress_controller.go

```diff
@@ -364,6 +405,26 @@ func (c *EgressController) deleteEgress(obj interface{}) {
 	klog.V(2).InfoS("Processed Egress DELETE event", "egress", klog.KObj(egress))
 }
 
+func (c *EgressController) addExternalIPPool(obj interface{}) {
+	pool := obj.(*crdv1b1.ExternalIPPool)
+	if pool.Spec.SubnetInfo == nil {
+		return
+	}
+	c.queue.Add(pool.Name)
```

Comment:
I feel like I may be asking a basic question, but how can we share the same workqueue between Egress resources and ExternalIPPool resources?